demisto_sdk/common/hook_validations/playbook.py

```diff
@@ -8,9 +8,22 @@ def is_valid_version(self):
         # type: () -> bool
         return self._is_valid_version()
 
-    def is_valid_playbook(self):  # type: () -> bool
+    def is_valid_playbook(self, is_new_playbook):  # type: (bool) -> bool
```

Comment:
We should have a default value for the arg

Reply:
Set to True